src/components/addMovieForm/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { createMovie } from '../../shared/api';
+import Dialog from '../dialog';
+import MovieForm from '../movieForm';
+
+const AddMovieForm: React.FC = () => {
+  const [showDialog, setShowDialog] = useState(true);
+  const navigate = useNavigate();
+
+  return (
+    <div>
+      {showDialog && (
+        <Dialog
+          data-testid='add-movie-dialog'
+          title='Add Movie'
+          onClose={() => {
+            setShowDialog(false);
+            navigate(-1);
+          }}
+        >
+          <MovieForm
+            onSubmit={async (movie) => {
+              setShowDialog(false);
+              await createMovie(movie);
```

Comment:
Please, add error handling and replace navigation -1 with smth less fragile. Check my comments for Edit Form.